src/aktualizr_secondary/CMakeLists.txt

```diff
@@ -112,40 +109,6 @@ add_test(NAME aktualizr_secondary_no_config_check_message
 set_tests_properties(aktualizr_secondary_no_config_check_message
                      PROPERTIES PASS_REGULAR_EXPRESSION "Config directory non-existent-config.toml does not exist.")
 
-add_test(NAME aktualizr_secondary_help_with_other_options
-         COMMAND aktualizr-secondary --help -c someconfig.toml)
-set_tests_properties(aktualizr_secondary_help_with_other_options
-                     PROPERTIES PASS_REGULAR_EXPRESSION "aktualizr-secondary command line options")
-
-add_test(NAME aktualizr_secondary_help_with_nonexistent_options
-         COMMAND aktualizr-secondary --help -somebadoption)
-set_tests_properties(aktualizr_secondary_help_with_nonexistent_options
-                     PROPERTIES PASS_REGULAR_EXPRESSION "aktualizr-secondary command line options")
-
-# Check aktualizr-secondary verbose config parsing output with debug loglevel.
-add_test(NAME aktualizr_secondary_log_debug
-         COMMAND aktualizr-secondary -c ${PROJECT_SOURCE_DIR}/tests/config/minimal.toml --loglevel=0)
-set_tests_properties(aktualizr_secondary_log_debug
-                     PROPERTIES PASS_REGULAR_EXPRESSION "Final configuration that will be used")
-# Check aktualizr-secondary silent config parsing output with default loglevel.
-# Note that the extra PASS is necessary to ignore the return code.
-add_test(NAME aktualizr_secondary_log_default
-         COMMAND aktualizr-secondary -c ${PROJECT_SOURCE_DIR}/tests/config/minimal.toml)
-set_tests_properties(aktualizr_secondary_log_default
-                     PROPERTIES FAIL_REGULAR_EXPRESSION "Final configuration that will be used"
-                                PASS_REGULAR_EXPRESSION "Aktualizr-secondary version")
-
-# Check aktualizr-secondary invalid logging levels.
-add_test(NAME aktualizr_secondary_log_invalid
-         COMMAND aktualizr-secondary -c ${PROJECT_SOURCE_DIR}/tests/config/aktualizr_secondary.toml --loglevel=6)
-set_tests_properties(aktualizr_secondary_log_invalid
-                     PROPERTIES PASS_REGULAR_EXPRESSION "Invalid log level")
-add_test(NAME aktualizr_secondary_log_negative
-         COMMAND aktualizr-secondary -c ${PROJECT_SOURCE_DIR}/tests/config/aktualizr_secondary.toml --loglevel=-1)
-set_tests_properties(aktualizr_secondary_log_negative
-                     PROPERTIES PASS_REGULAR_EXPRESSION "Invalid log level")
-
-
```

Comment:
I think it might be premature to remove all of these tests. Can we just mark them as `noptest` for now?

If we do remove them, we should also remove the checkmarks for these items in `actions.md`. I know that code is all a bit of a mess, but I'm not sure I'm ready to axe it all.

Reply:
Ok fair enough, this was a bit brutal. I had the idea of the "noptest" label after that, so that's why I've started removing stuff I did not like :).

Reply:
Now changed to use a softer method.

Still removed `aktualizr_secondary_cmdline_empty` which has the potential to run forever and is not really meaningful IMO.